rosdep/base.yaml

```
@@ -4431,6 +4431,8 @@ msgpack:
   fedora: [msgpack-devel]
   gentoo: [dev-libs/msgpack]
   ubuntu: [libmsgpack-dev]
+msr-tools:
+  ubuntu: [msr-tools]
```

Arch: https://archlinux.org/packages/community/x86_64/msr-tools/
Debian: https://packages.debian.org/search?suite=all&exact=1&searchon=names&keywords=msr-tools
Fedora/RHEL: https://src.fedoraproject.org/rpms/msr-tools#bodhi_updates
Gentoo: https://packages.gentoo.org/packages/sys-apps/msr-tools
OpenSUSE: https://software.opensuse.org/package/msr-tools

```suggestion
msr-tools:
  arch: [msr-tools]
  debian: [msr-tools]
  fedora: [msr-tools]
  gentoo: [sys-apps/msr-tools]
  opensuse: [msr-tools]
  rhel: [msr-tools]
  ubuntu: [msr-tools]
```